packages/kit/src/core/sync/write_tsconfig.js

```diff
@@ -46,7 +46,7 @@ export function write_tsconfig(config, cwd = process.cwd()) {
 	/** @param {string} file */
 	const config_relative = (file) => posixify(path.relative(config.outDir, file));
 
-	const include = ['ambient.d.ts'];
+	const include = ['ambient.d.ts', config_relative('vite.config.ts')];
```

Comment:
will this cause any issues if `vite.config.ts` is not present? do we need to list `vite.config.js` as well to make the `check` command check the `.js` version of the file?

Reply:
If something isn't present then that's no problem for TS. We don't need to include `vite.config.js` just because we include the TS version. It's included so ESBuild knows about it so it can bundle the config prior to execution (I believe that's how Vite gets away with that)